src/edu/wright/cs/raiderplanner/controller/ActivityController.java

```diff
@@ -175,7 +175,7 @@ public void validateDate() {
 	public void addTask() {
 		// Table items:
 		ObservableList<Task> list = FXCollections
-				.observableArrayList(MainController.getSpc().getCurrentTasks());
+					.observableArrayList(MainController.getSpc().getCurrentTasks());
```

Comment:
Please revert the change in indentation here. The continuation line should only be indented two tabs beyond the starting line.